frontend/server/src/Controllers/QualityNomination.php

```diff
@@ -608,12 +605,12 @@ private static function sendDemotionEmail(
      * @param int     $assignee  The user id of the person assigned to review
      *                           nominations.  May be null.
      *
-     * @return array The response.
+     * @return array{status: string, nominations: list<array{author: array{name: string, username: string}, contents?: array{before_ac: bool|null, difficultity: int|null, quality: int|null, rationale: null|string, reason: null|string, statements: array<string, string>|null, tags: list<string>|null}, nomination: string, nominator: array{name: string, username: string}, problem: array{alias: string, title: string}, qualitynomination_id: int, status: string, time: int, votes: array{time: int, user: array{name: string, username: string}, vote: int}[]}|null>} The response.
```

Comment:
```suggestion
     * @return array{status: string, nominations: list<array{author: array{name: string, username: string}, contents?: array{before_ac: bool|null, difficulty: int|null, quality: int|null, rationale: null|string, reason: null|string, statements: array<string, string>|null, tags: list<string>|null}, nomination: string, nominator: array{name: string, username: string}, problem: array{alias: string, title: string}, qualitynomination_id: int, status: string, time: int, votes: array{time: int, user: array{name: string, username: string}, vote: int}[]}|null>} The response.
```

welp, ese `difficultity` de dónde salió? hay que corregir código? D:

Reply:
Creo que sólo fue horror de dedo, se nos fue eso en algunas declaraciones de tipos nada más. Ya sustituí todos los lugares donde aparecía